src/index.ts

```diff
@@ -255,14 +255,15 @@ class Envoy {
     // ------------------------------------------------------------------------
 
 	// EXPOSED ENVOY EVENTS
-	
+
     // Envoy - New Hosted Agent Sign-up Sequence
     this.ws_server.register("holo/agent/signup", async ([hha_hash, agent_id]) => {
-      log.normal("Received sign-up request from Agent (%s) for HHA ID: %s", agent_id, hha_hash);      
+      log.normal("Received sign-up request from Agent (%s) for HHA ID: %s", agent_id, hha_hash);
+
       const failure_response = (new HoloError("Failed to create a new hosted agent")).toJSON();
 
       const anonymous_installed_app_id = hha_hash;
-      const registered_installed_app_id = `${hha_hash}:${agent_id}`;
+      const hosted_instance_app_id = `${hha_hash}:${agent_id}`;
```

Comment:
Why change this name?  Both the anonymous and registered instances of the app are hosted, which is why it was labelled with that distinction in mind.  I'm fine with changing the name, but don't want to lose this distinction.  What motivated this change?

Reply:
Only because this variable is in the sign-up function and the happ is not register"ed" yet

Reply:
Got it, so the idea of what is being registered is what is causing confusion.  I was using registered to signify that the agent is registered, rather than anonymous, but not that the happ was.  Can you think of a better way to relay that in a var name?

Reply:
Got it, so the idea of what is being registered is what is causing confusion.  I was using registered to signify that the agent is registered, rather than anonymous, but not that the happ was.  Can you think of a better way to relay that in a var name?

Reply:
I am only to suggestions

Reply:
Maybe let's go with `signed_in_installed_app_id` ?  What do you think?

Reply:
We settled on `hosted_agent_instance_app_id`